src/databricks/labs/remorph/resources/reconcile/dashboards/recon_details_schema.sql

```diff
@@ -0,0 +1,31 @@
+with tmp as (select
+recon_table_id,
+inserted_ts,
+explode(data) as schema_data
+from IDENTIFIER(:catalog || '.' || :schema || '.details' )
+where recon_type='schema'
+)
+select
+main.recon_id,
+main.source_table.`catalog` as source_catalog,
+main.source_table.`schema` as source_schema,
+main.source_table.table_name as source_table_name,
+CASE
+        WHEN COALESCE(MAIN.SOURCE_TABLE.CATALOG, '') <> '' THEN CONCAT(MAIN.SOURCE_TABLE.CATALOG, '.', MAIN.SOURCE_TABLE.SCHEMA, '.', MAIN.SOURCE_TABLE.TABLE_NAME)
+        ELSE CONCAT(MAIN.SOURCE_TABLE.SCHEMA, '.', MAIN.SOURCE_TABLE.TABLE_NAME)
+    END AS source_table,
+main.target_table.`catalog` as target_catalog,
+main.target_table.`schema` as target_schema,
+main.target_table.table_name as target_table_name,
+CONCAT(MAIN.TARGET_TABLE.CATALOG, '.', MAIN.TARGET_TABLE.SCHEMA, '.', MAIN.TARGET_TABLE.TABLE_NAME) AS target_table,
```

Comment:
Why are we repeating the same `Target Catalog, Schema, Table` Information again?

```suggestion
CONCAT_WS('.', MAIN.TARGET_TABLE.CATALOG, MAIN.TARGET_TABLE.SCHEMA, MAIN.TARGET_TABLE.TABLE_NAME) AS target_table,
```
Or

```suggestion
CONCAT_WS('.', TARGET_CATALOG, TARGET_SCHEMA, TARGET_TABLE_NAME) AS target_table,
```